config/initializers/upload_types.rb

```diff
@@ -0,0 +1,19 @@
+UPLOAD_TYPES = [
+    *CSV_TYPES_TABLES
```

Comment:
is this list going to be more than `CSV_TYPES_TABLES`?  `dup` could work here.

Reply:
Yes, in https://github.com/department-of-veterans-affairs/gibct-data-service/pull/751 there is another array be splatted into this

I'm creating arrays that are merged because while both arrays will be displayed in same location the behavior of the "upload" button is different